build-cassandra-bin.bash

```diff
@@ -24,7 +24,7 @@ fi
 # VERSION SETTINGS
 CASSANDRA_VERSION="3.0.10"
 METRICS_INTERFACE_VERSION="3" # Cassandra 2.2+ uses metrics3, while <= 2.1 uses metrics2.
-STATSD_REPORTER_VERSION="4.1.2"
+STATSD_REPORTER_VERSION="4.1.2-SNAPSHOT"
```

Comment:
Actually, 3.1.0 has been there since Cassandra 2.2, but it wasn't a problem until Cassandra 3.0 (not sure why). I'll add a comment.